website/docs/installation/upgrade.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Upgrade authentik
+---
+
+Upgrading to the latest version of authentik, whether a new major release or a patch, involves running a few commands to pull down the latest images and then restart the servers and databases.
+
+For information about a specific version, refer to our [Release Notes](../releases/).
```

Comment:
We need to be more specific about the fact that it's not optional to read to release notes.